doc/modules/cross_validation.rst

```diff
@@ -719,8 +719,9 @@ group information can be used to encode arbitrary domain specific pre-defined
 cross-validation folds.
 
 Each training set is thus constituted by all the samples except the ones
-related to a specific group. This is basically the same as
-:class:`LeavePGroupsOut` with ``n_groups=1``.
+related to a specific group. This is the same as :class:`LeavePGroupsOut` with
+`n_groups=1` and the same as :class:`GroupKFold` with `n_splits` equal to the
```

Comment:
See suggested change: https://github.com/scikit-learn/scikit-learn/pull/24104#discussion_r941576489

I think only single backticks now render as code blocks, see rendered docs: https://output.circle-artifacts.com/output/job/27ea1399-baac-45c4-8d8e-f3047890e8b2/artifacts/0/doc/modules/cross_validation.html#leave-one-group-out

I think single back tick function was changed here: https://github.com/scikit-learn/scikit-learn/pull/14817

Reply:
Oh, good to know. FWIW the double backtick still works also as in line 612 of the `.rst` file.

Reply:
Yes, it's been suggested to me in another review to use single backtick but not sure if there is overall consensus on this.

Reply:
Single backticks and double backticks both work and do the same thing. These days, I prefer new contributions to use single backticks because:

1. Many contributors are coming from markdown which uses a single backticks. (There is less cycles when reviewing and asking contributors to use double backticks.)
2. Single backticks have less visible clutter when looking at a docstring in a Python REPR or Jupyter with `my_function?`.

Note that single backticks is configured with Sphinx's [default_role](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-default_role) which is `None` by default. From what I've seen, libraries choose the default that best fits their community. (There is no consistent `default_role` between libraries).

Reply:
Agree with the visual clutter!